src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexNode.cs

```diff
@@ -1344,17 +1552,15 @@ public string Description()
                 _ => $"(unknown {Type})"
             };
 
-            var argSb = new StringBuilder().Append(typeStr);
-
-            if ((Options & RegexOptions.ExplicitCapture) != 0) argSb.Append("-C");
-            if ((Options & RegexOptions.IgnoreCase) != 0) argSb.Append("-I");
-            if ((Options & RegexOptions.RightToLeft) != 0) argSb.Append("-L");
-            if ((Options & RegexOptions.Multiline) != 0) argSb.Append("-M");
-            if ((Options & RegexOptions.Singleline) != 0) argSb.Append("-S");
-            if ((Options & RegexOptions.IgnorePatternWhitespace) != 0) argSb.Append("-X");
-            if ((Options & RegexOptions.ECMAScript) != 0) argSb.Append("-E");
+            var sb = new StringBuilder().Append(typeStr);
```

Comment:
nit `var sb = new StringBuilder(typeStr);`

Reply:
They're actually different.  But this is debug code, perf doesn't matter, I can change it.

Reply:
They are? Only in the details of the backing storage size?

Reply:
> Only in the details of the backing storage size?

Right.